src/test/ui/issue-48636.stderr

```diff
@@ -0,0 +1,13 @@
+error[E0585]: found a documentation comment that doesn't document anything
+  --> $DIR/issue-48636.rs:13:5
+   |
+LL |     x: u8
+   |        -- help: missing comma here: `,`
```

Comment:
As pointed out above, this suggestion should be pointing at the span _after_ `u8`.